src/dropdown.ts

```diff
@@ -348,7 +348,7 @@ export default class GlideCoreDropdown
   variant?: 'quiet';
 
   @property({ reflect: true })
-  readonly version = packageJson.version;
+  readonly version: string = packageJson.version;
```

Comment:
Missed a couple. 

`version` is unique. But I'll create a lint rule for adding an explicit type to attributes whose initializer is an expression.